src/ui/activity-iframe-view-test.js

```diff
@@ -97,6 +106,30 @@ describes.realWin('ActivityIframeView', {}, (env) => {
       expect(activityIframePort.whenReady).to.have.been.calledOnce;
     });
 
+    it('should disallow scrolling within dialogs', async () => {
```

Comment:
nit: can omit "should" for brevity, so the test description would read something like "disallows scrolling within dialogs"

Reply:
Good call